index.html

```diff
@@ -2747,21 +2760,34 @@
             </li>
           </ol>
           <p>
+            All <a data-lt="nested browsing context">nested browsing
+            contexts</a> <a data-lt=
+            "creating a new browsing context">created</a> by the presented
+            document, i.e. that have the <a>receiving browsing context</a> as
+            their <a data-lt="ancestor browsing context">top-level
+            ancestor</a>, MUST also have restrictions 2-4 above. All of these
```

Comment:
Given the definition of top-level browsing context in HTML, I believe you can simply say "their <a>top-level browsing context</a>" here and drop the reference to "ancestor browsing context" (and drop it from the list of terms too).